content/docs/identity-providers/azure.mdx

```diff
@@ -1,156 +1,177 @@
 ---
-title: Azure Active Directory
-sidebar_label: Azure AD
+title: Entra ID (Azure Active Directory)
```

Comment:
Based on https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-id it sounds like the full name is "Microsoft Entra ID".

It might also be good to put "formerly" in the title as well as the description?

```suggestion
title: Microsoft Entra ID (formerly Azure Active Directory)
```